runtime/runtime.ts

```diff
@@ -402,52 +402,47 @@ function lookup(table: any, index: number): number {
 }
 
 function abs(a: number): number {
-  a = +a;
-  return Math.abs(a);
+  return Math.abs(+a);
```

Comment:
I don't love this refactoring of inlining the number cast (it feels like it is easier to mistake as an accidental typo), but don't feel its a blocker